docs/wiki/[Examples]-Create-and-Assign-Custom-RBAC-Roles.md

```diff
@@ -0,0 +1,128 @@
+## Overview
+
+This page describes how to use the module to create your own custom RBAC roles and assign them at the appropriate scopes within your Enterprise-Scale deployment.
+
+In this example, we will create a custom RBAC role called "Reader Support Tickets" which will allow members to read everything in a subscription and to also open support tickets.
+
+We will then assign the new RBAC Role at the `es_landingzones` scope.
+
+We will update the built-in configuration with the new custom RBAC role by following these steps:
+
+- Create the role definition file
+
+- Save the new role at the `es_root` scope
+
+- Assign the role at the `es_landingzones` scope
+
+The files we will use are:
+
+- [lib/role_definitions/role_definition_es_reader_support_tickets.tmpl.json](#librole_definitionsrole_definition_es_reader_support_ticketstmpljson)
+- [lib/archetype_extension_es_root.tmpl.json](#libarchetype_extension_es_roottmpljson)
+- [lib/archetype_extension_es_landing_zones.tmpl.json](#libarchetype_extension_es_landing_zones_tmpljson)
+
+>IMPORTANT: To allow the declaration of custom or expanded templates, you must create a custom library folder within the root module and include the path to this folder using the `library_path` variable within the module configuration. In our example, the directory is `/lib`.
```

Comment:
Suggest referring to the lib directory as `lib` or `./lib`. Using `/lib` is an absolute path that might confuse people.

Reply:
Updated to `lib`.